lib/absinthe/resolution/helpers.ex

```diff
@@ -165,6 +165,13 @@ defmodule Absinthe.Resolution.Helpers do
       field :organization, :organization do
         resolve dataloader(Accounts, :organization, use_parent: false)
       end
+
+      field(:subscription_active, non_null(:boolean), resolve: dataloader(
```

Comment:
Let's pick an example that doesn't involve the word "subscription", I'm concerned it will confuse people who will think this has something to do with GraphQL subscriptions.

Reply:
Good point